IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_RailwayFacilityPartTypeRailwayCrossing/DocPropertySet.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_RailwayFacilityPartTypeRailwayCrossing" UniqueId="7f547f91-6dd0-161c-9824-1f608dd660d9" ApplicableType="IfcFacilityPart" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
```

Comment:
Consistent Naming Pset_FacilityPartTypeRailwayCrossing or Pset_RailwayPartTypeRailwayCrossing

RailwayPartType is the enum name or FacilityPart is the entity name.

Reply:
See above